Library/Homebrew/cli/args.rb

```diff
@@ -167,6 +180,19 @@ def devel
         (args_parsed && devel?) || cmdline_args.include?("--devel")
       end
 
+      def build_universal
+        (args_parsed && universal?) || cmdline_args.include?("--universal")
+      end
+
+      def build_bottle
+        (args_parsed && build_bottle?) || cmdline_args.include?("--build-bottle")
+      end
+
+      def build_from_source
+        (args_parsed && (build_from_source? || s?)) || cmdline_args.include?("--build-from-source") ||
+          cmdline_args.include?("-s")
```

Comment:
```suggestion
        return true if args_parsed && (build_from_source? || s?) 
        
        cmdline_args.include?("--build-from-source") || cmdline_args.include?("-s")
```

Reply:
Seems clearer and more readable. Thanks!